ci-automation/release.sh

```diff
@@ -175,7 +175,23 @@ function copy_from_bincache_to_bucket() {
       docker.io/rclone/rclone:1.71.1 \
         --config "/opt/rclone.conf" \
         sync \
-        --http-url "https://${BUILDCACHE_SERVER}/images/${arch}/${version}" :http: "r2:flatcar/${channel}/${arch}-usr/${version}" || { echo "ERROR: Skipping bucket copy due to failure" ; }
+        --http-url "https://${BUILDCACHE_SERVER}/images/${arch}/${version}" :http: "r2:flatcar/${channel}/${arch}-usr/${version}" || { echo "ERROR: Skipping images copy to bucket due to failure" ; }
+
+    echo "Copying the binary packages from bincache to CloudFlare bucket"
+    docker run --rm --net host \
+      -v "${rclone_configuration_file}:/opt/rclone.conf:ro" \
+      docker.io/rclone/rclone:1.71.1 \
+        --config "/opt/rclone.conf" \
+        sync \
+        --http-url "https://${BUILDCACHE_SERVER}/boards/${arch}-usr/${version}" :http: "r2:flatcar/mirror/boards/${arch}-usr/${version}" || { echo "ERROR: Skipping binary packages copy to bucket due to failure" ; }
+
+    echo "Copying SDK and packages containers from bincache to CloudFlare bucket"
+    docker run --rm --net host \
+      -v "${rclone_configuration_file}:/opt/rclone.conf:ro" \
+      docker.io/rclone/rclone:1.71.1 \
+        --config "/opt/rclone.conf" \
+        sync \
+        --http-url "https://${BUILDCACHE_SERVER}/containers/${version}" :http: "r2:flatcar/mirror/containers/${version}" || { echo "ERROR: Skipping containers copy (SDK / packages) to bucket due to failure" ; }
     )
```

Comment:
We need one more, or?
`/sdk/amd64/${VERSION}` but only when version equals SDK version (i.e., a new SDK was built)

Reply:
Ah right, maybe we can add it to the section where we push the container to the github registry.